spec/services/content_items_service_spec.rb

```diff
@@ -26,6 +26,16 @@
       expect(result).to match_array([:a])
     end
 
+    it "doesn't yield empty content items" do
```

Comment:
This is a good test 👍  Would you mind updating the copy to something that make it explicit that it is not yielding `nil` responses from the `ContentStore`